src/roster/templates/roster/partial/student_details_reading_details.html

```diff
@@ -67,9 +67,21 @@ <h4 class="h2 mb-0">Reading details</h4>
                                 {% endif %}
                             </th>
                             <td>{{ book.hours }} <span aria-hidden="true">hrs</span><br><span class="sr-only">hours</span> ({{ book.view_count }} visits)</td>
-                            <td style="white-space:nowrap;">{{ book.last_view | naturalday:"D, M d" }}</td>
+                            <td style="white-space:nowrap;">
+                                {% if book.last_view %}
+                                    {{ book.last_view | naturalday:"D, M d" }}
+                                {% else %}
+                                    Unknown
+                                {% endif %}
+                            </td>
```

Comment:
It might work here to just say `{ book.last_view | naturalday:"D, M d" | default:"Unknown" }}

Reply:
It does!  That makes it cleaner.